src/asm/rgbasm.5

```diff
@@ -1490,31 +1491,50 @@ This is the only way of accessing the value of arguments from 10 to 256.
 .Ic SHIFT
 can optionally be given an integer parameter, and will apply the above shifting that number of times.
 .Ss Printing things during assembly
-The next four commands print text and values to the standard output.
+The
+.Ic PRINT
+and
+.Ic PRINTLN
+commands print text and values to the standard output.
 Useful for debugging macros, or wherever you may feel the need to tell yourself some important information.
 .Bd -literal -offset indent
-PRINTT "I'm the greatest programmer in the whole wide world\[rs]n"
-PRINTI (2 + 3) / 5
-PRINTV $FF00 + $F0
-PRINTF MUL(3.14, 3987.0)
+PRINT "Hello world!\[rs]n"
+PRINTLN "Hello world!"
+PRINT _NARG, " arguments\[rs]n"
+PRINTLN "sum: ", 2+3, " product: ", 2*3
+PRINTLN "Line #", __LINE__
+PRINTLN STRFMT("E = %f", 2.718)
 .Ed
 .Bl -inset
+.It Ic PRINT
+prints out each of its comma-separated arguments.
+Numbers are printed as unsigned uppercase hexadecimal with a leading
+.Ic $ .
+For different formats, use
+.Ic STRFMT .
+.It Ic PRINTLN
+prints out each of its comma-separated arguments, if any, followed by a line feed
+.Pq Ql \[rs]n .
+.El
+.Pp
+Four other printing commands have been deprecated. Use
```

Comment:
We avoid mentioning deprecations in the docs, especially since the deprecation warning tells what the alternative is. (If the user disabled the warning, or doesn't pay attention to it, they'll be responsible!)

Since the other symbols have been deprecated, they can be purged from the documentation as well. (If someone still needs to check their behavior, they can refer to the previous version's documentation, since the behavior hasn't been changed.)

Reply:
Oh; `*` comments, colon-less labels, and `GLOBAL` are mentioned as deprecated.

Reply:
I see. They should be removed soon, anyway.

Reply:
Removed the `PRINT[TFVI]` docs.